### ISSUE TITLE
Tutorial for `stack`

### ISSUE THREAD

**Issue by parsonsmatt** [parsonsmatt/hasktuts#1]
There have been a few requests for tutorials and documentation on using `stack`. There's nothing on dohaskell either, so that's a good place to start.

**Comment by tredontho** [comment]
Maybe it's not appropriate for a general `stack` tutorial, but I had a heck of a time trying to get everything working with ghc-mod and atom when using `stack` on Windows... then again, it seems like most things haskell on Windows is a bit of a crapshoot.

**Comment by duplode** [comment]
Michael Snoyman is working on an [official guide](https://github.com/commercialhaskell/stack/blob/master/GUIDE.md); however, that seems a little too heavy to be something that you'd suggest as a first reading to a new Haskeller.

**Comment by parsonsmatt** [comment]
Yeah -- I'd like for the content here to be beginner friendly introductory materials to get people started, rather than complete/comprehensive documentation.

**Comment by CGenie** [comment]
How about https://www.fpcomplete.com/blog/2015/08/new-in-depth-guide-stack ?

**Comment by duplode** [comment]
@CGenie That is (the final version of) the official guide I linked to above.

**Comment by tredontho** [comment]
Hmm, I suppose you're right.  Perhaps that would be better suited to a "Getting started with ${editor}" or "Getting started on ${OS}" type of tutorial.
